lib/experiments-page.php

```diff
@@ -84,6 +84,17 @@ function gutenberg_initialize_experiments_settings() {
 			'id'    => 'gutenberg-full-site-editing-demo',
 		)
 	);
+	add_settings_field(
```

Comment:
This is unrelated to this PR, probably inadvertently added in a rebase. Will remove.

Reply:
Removed in https://github.com/WordPress/gutenberg/pull/20047/commits/6f1e87b1e041921dbc9e5a0582dd52920db71fbd